<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LkSetting</class>
 <widget class="QDialog" name="LkSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>541</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>视频</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>一般设置</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>FPS：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_LkFPS">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>码率：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkMl">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_LkZxh">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>点击推送后自动最小化</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>分辨率：</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_LkXssb">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>显示鼠标</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkFbl">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_LkAero">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>启动时禁用Aero效果</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_LkSaveFile">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>将直播视频保存在本地</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>保存路径：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_LkSavePath">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>380</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SelectPath">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>380</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_LkDesktop">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>添加桌面源时最小化</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_More">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>高级</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>高级</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>视频</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>155</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>关键帧数目：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkCPUys">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkGjzsm">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>275</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>码率类型：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>编码器：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>195</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x264 CPU预设：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkPSYTj">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x264</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkBxbmzs">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_LkBgjz">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>使用 B 幀作为关键帧</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>PSY调节：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>品质：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>235</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>并行编码帧数：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkPzwj">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkMllx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>编码配置文件：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_LkPz">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>261</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>音频</string>
     </property>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>设备：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkSb">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkBmgs">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>编码格式：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkPd">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>频道：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkCyl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>155</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>采样率：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_LkBl">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>195</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>别率：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>561</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>网络</string>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>流媒体地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_LkURL">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>19</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>LkSetting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>LkSetting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
